src/renderers/shaders/ShaderChunk/shadowmap_vertex.glsl

```diff
@@ -2,7 +2,36 @@
 
 	for( int i = 0; i < MAX_SHADOWS; i ++ ) {
 
-		vShadowCoord[ i ] = shadowMatrix[ i ] * worldPosition;
+		#if defined(POINT_LIGHT_SHADOWS)
+
+			// if shadowDarkness[ i ] < 0.0, that means we have a point light with a cube
+			// shadow map
+			if( shadowDarkness[ i ] < 0.0 ) {
+
+				// When we have a point light, the @shadowMatrix uniform is used to store
+				// the inverse of the view matrix, so that we can get the world-space
+				// position of the light.
+
```

Comment:
I am not a bit fan of repurposing, in this case, and elsewhere in this PR, but of course, that is a matter of opinion.

In this case, the repurposing is not necessary, as we already have access to the `viewMatrix`. We have used a similar trick previously in `common.glsl` to transform direction vectors by the inverse of the view matrix.

See https://en.wikibooks.org/wiki/GLSL_Programming/Applying_Matrix_Transformations, and in particular the section _Transforming Points with the Inverse Matrix_. 

@tschw Can you confirm?